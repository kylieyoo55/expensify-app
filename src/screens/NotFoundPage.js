import React from 'react';
import { NavLink} from 'react-router-dom';

const NotFoundPage = ()=>(
    <div>
        404 <NavLink to='/'>GO HOME</NavLink>
    </div>
)

export default NotFoundPage;