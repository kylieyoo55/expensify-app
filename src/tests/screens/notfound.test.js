import React from 'react';
import {shallow} from 'enzyme';
import NotFoundPage from '../../screens/NotFoundPage'
import toJSON from 'enzyme-to-json';

test('should render not found correctly',()=>{
    const wrapper=shallow(<NotFoundPage />);
    expect(toJSON(wrapper)).toMatchSnapshot();
})