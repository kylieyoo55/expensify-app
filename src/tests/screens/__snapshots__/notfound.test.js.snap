// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not found correctly 1`] = `
<div>
  404 
  <NavLink
    to="/"
  >
    GO HOME
  </NavLink>
</div>
`;
